Sources/PubNub/Networking/HTTPRouter.swift

```diff
@@ -210,7 +212,7 @@ extension HTTPRouter {
       URLQueryItem(name: "uuid", value: configuration.uuid)
     ]
     // Add PAM key if needed
-    if pamVersion != .none, let authKey = configuration.authKey {
+    if pamVersion != .none, let authKey = configuration.token ?? configuration.authKey {
```

Comment:
Is there ever a reason why the user would want to be able to use the `authKey` over the `token`, or is this part of the PAM requirements?

Reply:
Yep:

> Everywhere the SDK supports passing an auth key we should also allow passing a token and give tokens priority over auth keys when both are present